bin/migration-scripts/0.14.26.js

```diff
@@ -0,0 +1,53 @@
+export const description = 'v0.14.26';
+
+export default (uniPath, content) => {
+  if (!content?.blocks) {
+    return content;
+  }
+
+  for (const block of content.blocks) {
+    adjustCarouselRecommendationCardBlock(block);
+    adjustCarouselProgramsCardBlock(block);
+  }
+
+  return { ...content };
+};
+
+function adjustCarouselProgramsCardBlock(block) {
+  if (block.blocks) {
+    for (const childBlock of block.blocks) {
+      adjustCarouselProgramsCardBlock(childBlock);
+    }
+  }
+
+  const content = block?.content;
+  if (content?.list?.length === 0) {
```

Comment:
Если придет `undefined`, то будет сравнение `undefined === 0`

Тогда лучше так:


```suggestion
  if (!content?.list?.length) {
```